### ISSUE TITLE
Convert improper epytext to proper epytext

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8497]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)| @acabhishek942 reported|
|-|-|
|Trac ID|trac#8497|
|Type|enhancement|
|Created|2016-06-24 08:40:13Z|

Checking out the trunk and running
```
bin/admin/build-apidocs . apidocs

```

reveals docstrings that are improper epytext

```
these 18 objects' docstrings are not proper epytext:
    twisted.internet.interfaces.IReactorSocket.adoptDatagramPort
    twisted.names.server.DNSServerFactory
    twisted.names.server.DNSServerFactory.__init__
    twisted.names.dns._OPTHeader
    twisted.names.dns._OPTVariableOption
    twisted.names.dns._EDNSMessage._fromMessage
    twisted.names.dns._EDNSMessage.__init__
    twisted.persisted.styles._pickleFunction
    twisted.web._http2.H2Stream.unregisterProducer
    twisted.web._http2.H2Stream.stopProducing
    twisted.web._http2.H2Stream.pauseProducing
    twisted.web._http2.H2Stream.resumeProducing
    twisted.conch.ssh.transport.SSHCiphers
    twisted.conch.ssh.keys._getPersistentRSAKey
    twisted.python.logfile.BaseLogFile._openFile
    twisted.trial._dist.workerreporter.WorkerReporter
    twisted.trial._dist.workerreporter.WorkerReporter._getTodoReason
    twisted.mail.smtp.ESMTPClient.registerAuthenticator
```

<details><summary>Searchable metadata</summary>

```
trac-id__8497 8497
type__enhancement enhancement
reporter__acabhishek942 acabhishek942
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy__documentation easy, documentation
time__1466757613735600 1466757613735600
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)|@acabhishek942 removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_2"></a>|@acabhishek942 commented|
|-|-|

Link to PR - https://github.com/twisted/twisted/pull/213

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc commented|
|-|-|

Please see comments in: https://github.com/twisted/twisted/pull/213
Please fix those comments and push them to your branch, and I will review again.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 36d1cd4f5ebf692bd9a00fa1bee956187c845799
```
#!CommitTicketReference repository="" revision="36d1cd4f5ebf692bd9a00fa1bee956187c845799"
Merge 8497-acabhishek942-doc: Fix epydoc errors

Author: acabhishek942
Reviewer: rodrigc
Fixes: #8497
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
